scipy/stats/tests/test_distributions.py

```diff
@@ -2470,6 +2470,17 @@ def test_logcdf_logsf(self):
         logsf = stats.invgauss.logsf(110, 1.05)
         assert_allclose(logsf, -56.1467092416426)
 
+    # from mpmath import mp
+    # mp.dps = 100
+    # mu = mp.mpf(1e-2)
+    # ref = (1/2 * mp.log(2 * mp.pi * mp.e * mu**3)
+    #        - 3/2* mp.exp(2/mu) * mp.e1(2/mu))
+    @pytest.mark.parametrize("mu, ref", [(1e-2, -5.496279615262233),
+                                         (1e8, 3.3244822568873474)])
+                                         (1e100, 3.3244828013968899)])
```

Comment:
We get precisely the same results with mpmath numerical integration:
```python3
import numpy as np
from mpmath import mp
mp.dps = 100

def _logpdf(x, mu):
    # SciPy's definition of _logpdf
    return -0.5 * mp.log(2 * mp.pi) - 1.5 * mp.log(x) - ((x - mu) / mu) ** 2 / (2 * x)

def logpdf(x, mu, loc, scale):
    return _logpdf((x - loc)/scale, mu) - mp.log(scale)

@np.vectorize
def entropy(mu, loc=0.0, scale=1.0):
    mu, loc, scale = mp.mpf(mu), mp.mpf(loc), mp.mpf(scale)
    def integrand(x):
        logf = logpdf(x, mu, loc, scale)
        f = mp.exp(logf)
        return -f * logf
    return mp.quad(integrand, (loc, loc+mu, np.inf))

entropy(0.01)

# array(mpf('-5.496279615262233314139570946311748385591116312369781831539906340907631749051583759839841056064307524042'), dtype=object)
```
It agrees with the formula in all but the last two of the 103 digits.

Reply:
Nice trick with @np.vectorize! I actually computed those values using `mp.quad` too, just forgot to add that to the comments.  Sry that you had to implement it again.

Reply:
No, I already had this. Actually, I'll open a PR to add an `mpmath` distribution interface to the top of `test_distributions.py` so we don't need to repeat the same sort of code everywhere.